runtime/src/main/scala/com/ing/baker/runtime/scaladsl/ProcessState.scala

```diff
@@ -17,9 +17,13 @@ import scala.collection.JavaConverters._
 case class ProcessState(
     processId: String,
     ingredients: Map[String, Value],
-    eventNames: Seq[String])
+    events: Seq[RuntimeEvent])
```

Comment:
As discusses: To prevent duplication of the ingredients data, we will keep our old data structure, with the addition of tuples with timestamps for the eventNames. And at the proto and javadsl level, make an extra message/class for it.